apps/openchallenges/mcp-server/project.json

```diff
@@ -72,6 +72,13 @@
         "command": "trivy image ghcr.io/sage-bionetworks/{projectName}:local --quiet",
         "color": true
       }
+    },
+    "inspect": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "npx -y @modelcontextprotocol/inspector node build/index.js",
```

Comment:
@rrchai We should add this package as a dev package to the workspace `package.json`. You should then be able to run `inspector node build/index.js` inside this task.

Reply:
Nice initiative!

Reply:
installed the package via `pnpm add -D -w @modelcontextprotocol/inspector`